src/Domain/Helper/FilesFinder.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Domain\Helper;
+
+use Symfony\Component\Finder\Finder;
+
+/**
+ * @internal
+ */
+final class FilesFinder
```

Comment:
The goal is to return a list of file inside a list of directory. A `Directory::find` helper sounds for me like returning a list of directories ?

Maybe it would be called `Files::findInDirectories` ?

Reply:
Ah yeah, good point 👍  
That looks fine with that name. 

Just to come with another alternative would be a builder. 
`Files::inDirectory($dir)->find([...]);`

However I think calling it `Files::findInDirectories` would also work great here 😃

Reply:
After rethinking it, I just renamed it Files. In effect, in find method, we can pass a list of relative file pathname also, not only directory.